tests/snmpsim/snr.snmprec

```diff
@@ -0,0 +1,1261 @@
+ All rights reserved
```

Comment:
It looks like these are encoded incorrectly...  Delete these text lines.
I assume they are part of the sysDescr but got messed up somehow.
If you send me the output of snmpget 1.3.6.1.2.1.1.1.0 from the device, I'll encode it correctly.

Reply:
1.3.6.1.2.1.1.1.0
SNMPv2-MIB::sysDescr.0 = STRING: SNR-S2990X-24FQ Device, Compiled Jan 10 13:47:05 2019
 SoftWare Version 7.5.3.2(R0010.0044)
 BootRom Version 7.5.6
 HardWare Version 1.0.1
 Serial No.:SW080710J513000055
 Copyright (C) 2019 NAG LLC
 All rights reserved